sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -4,11 +4,17 @@
 
 ### Breaking changes
 
-- Property `Type` renamed to `ValueType` on `FieldValue`.
+- Renamed the `FieldValue` property `Type` to `ValueType`.
```

Comment:
Putting the verb first, as suggested by our native English speaker representative (@annelo-msft).